tests/integration/cli/detection/test_detection.py

```diff
@@ -56,6 +56,26 @@
     "train_params": ["params", "--learning_parameters.num_iters", "2", "--learning_parameters.batch_size", "4"],
 }
 
+semisl_args = {
```

Comment:
I didn't define new args for test, just appended keys for semi in copied args dict.